http-gateway/web/public/web_configuration.json

```diff
@@ -1,15 +1,17 @@
 {
-  "authority": "",
+  "authority": "https://auth.plgd.cloud/realms/shared",
```

Comment:
web_configuration.json shouldn't be needed at all. Can you delete it and make use only of `window.location.origin`?